### PR TITLE
du: add German translation

### PR DESCRIPTION
**Pull request by k-stz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/du.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -sh {{Pfad/zum/Verzeichnis}}`
+
+- Liste die Größe eines Verzeichnisses und seiner Dateien in für Menschen lesbaren Einheiten:
+
+`du -ah {{Pfad/zum/Verzeichnis}}`
+
+- Auflistung der menschenlesbaren Größen eines Verzeichnisses und aller Unterverzeichnisse, bis zu einer Tiefe von `N` Ebenen:
+
+`du -h --max-depth=N {{Pfad/zum/Verzeichnis}}`
```
Comment (navarroaxel) [inline pages.de/common/du.md:24]:
```suggestion
`du -h --max-depth=N {{pfad/zu/verzeichnis}}`
```

File: pages.de/common/du.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# du
+
+> Disk usage: Plattenplatzverbrauch von Dateien und Verzeichnissen ermitteln.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/du>.
+
+- Größe von Verzeichnissen und Unterverzeichnissen in den gegebenen Einheiten (B/KB/MB) auflisten:
+
+`du -{{b|k|m}} {{Pfad/zum/Verzeichnis}}`
```
Comment (navarroaxel) [inline pages.de/common/du.md:8]: We use `pfad/zu/verzeichnis`
```suggestion
`du -{{b|k|m}} {{pfad/zu/verzeichnis}}`
```

File: pages.de/common/du.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Liste die Größe von Verzeichnissen und Unterverzeichnissen in für Menschen lesbaren Einheiten auf (d. h. die automatische Auswahl der geeigneten Einheit für jede Größe):
+
+`du -h {{Pfad/zum/Verzeichnis}}`
```
Comment (navarroaxel) [inline pages.de/common/du.md:12]:
```suggestion
`du -h {{pfad/zu/verzeichnis}}`
```

File: pages.de/common/du.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Zeige die Größe eines Verzeichnisses in für Menschen lesbaren Einheiten:
+
+`du -sh {{Pfad/zum/Verzeichnis}}`
+
+- Liste die Größe eines Verzeichnisses und seiner Dateien in für Menschen lesbaren Einheiten:
+
+`du -ah {{Pfad/zum/Verzeichnis}}`
```
Comment (navarroaxel) [inline pages.de/common/du.md:20]:
```suggestion
`du -ah {{pfad/zu/verzeichnis}}`
```

File: pages.de/common/du.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -{{b|k|m}} {{Pfad/zum/Verzeichnis}}`
+
+- Liste die Größe von Verzeichnissen und Unterverzeichnissen in für Menschen lesbaren Einheiten auf (d. h. die automatische Auswahl der geeigneten Einheit für jede Größe):
+
+`du -h {{Pfad/zum/Verzeichnis}}`
+
+- Zeige die Größe eines Verzeichnisses in für Menschen lesbaren Einheiten:
+
+`du -sh {{Pfad/zum/Verzeichnis}}`
```
Comment (navarroaxel) [inline pages.de/common/du.md:16]:
```suggestion
`du -sh {{pfad/zu/verzeichnis}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.de/common/du.md (hunk 5 above)
Comment (k-stz) [reply pages.de/common/du.md]: fixed with latest commit

File: pages.de/common/du.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# du
+
+> Disk usage: Plattenplatzverbrauch von Dateien und Verzeichnissen ermitteln.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/du>.
+
+- Größe von Verzeichnissen und Unterverzeichnissen in den gegebenen Einheiten (B/KB/MB) auflisten:
+
+`du -{{b|k|m}} {{pfad/zu/verzeichnis}}`
+
+- Liste die Größe von Verzeichnissen und Unterverzeichnissen in für Menschen lesbaren Einheiten auf (d. h. die automatische Auswahl der geeigneten Einheit für jede Größe):
```
Comment (pixelcmtd) [inline pages.de/common/du.md:10]:
```suggestion
- Liste die Größe von Verzeichnissen und Unterverzeichnissen in menschenlesbaren Einheiten auf (d.h. automatische Auswahl der geeigneten Einheit für jede Größe):
```

File: pages.de/common/du.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -h {{pfad/zu/verzeichnis}}`
+
+- Zeige die Größe eines Verzeichnisses in für Menschen lesbaren Einheiten:
```
Comment (pixelcmtd) [inline pages.de/common/du.md:14]:
```suggestion
- Zeige die Größe eines einzelnen Verzeichnisses in menschenlesbaren Einheiten:
```

File: pages.de/common/du.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -sh {{pfad/zu/verzeichnis}}`
+
+- Liste die Größe eines Verzeichnisses und seiner Dateien in für Menschen lesbaren Einheiten:
```
Comment (pixelcmtd) [inline pages.de/common/du.md:18]:
```suggestion
- Liste die Größe eines Verzeichnisses und seiner Dateien in menschenlesbaren Einheiten:
```

File: pages.de/common/du.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -ah {{pfad/zu/verzeichnis}}`
+
+- Auflistung der menschenlesbaren Größen eines Verzeichnisses und aller Unterverzeichnisse, bis zu einer Tiefe von `N` Ebenen:
```
Comment (pixelcmtd) [inline pages.de/common/du.md:22]:
```suggestion
- Liste die menschenlesbaren Größen eines Verzeichnisses und aller Unterverzeichnisse, bis zu einer Tiefe von `N` Ebenen:
```

File: pages.de/common/du.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`du -h --max-depth=N {{pfad/zu/verzeichnis}}`
+
+- Liste die Größe aller `.jpg`-Dateien in Unterverzeichnissen des aktuellen Verzeichnisses auf, in für menschenlesbaren Einheiten und zeige am Ende die kumulierte Gesamtsumme an:
```
Comment (pixelcmtd) [inline pages.de/common/du.md:26]:
```suggestion
- Liste die menschenlesbare Größe aller `.jpg`-Dateien in Unterverzeichnissen des aktuellen Verzeichnisses auf und zeige am Ende die kumulierte Gesamtsumme an:
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @k-stz

**Pull request merged**